### DISCUSSION TITLE
Seed Discussions by moving select items from Ideas repository

### DISCUSSION THREAD

**Discussion by jqnatividad** [ckan/ckan#5991] [Ideas]
Now that we're using GitHub Discussions :tada: (closes https://github.com/ckan/ideas/issues/262 and https://github.com/ckan/ideas/issues/272 ), let's seed it by moving select items from https://github.com/ckan/ideas, with the ultimate goal of archiving the Ideas repo once all still valid open items are moved here.

cc @ckan/core @Starl3n @rufuspollock @monikappv @amercader @anuveyatsu @wardi

**Comment by rufuspollock** [comment]
Big 👍 @jqnatividad and let's start doing that.

**Comment by amercader** [comment]
For now I've marked the ideas repo as deprecated:

I tested the process migrating this recent idea, and what I think would work best is 
1. Transfer the issue to the main ckan repo
2. Convert to Discussion:

https://github.com/ckan/ckan/discussions/6022

This leaves a closed and locked issue around in this repo but I don't think that's a big deal.
